src/lib.rs

```diff
@@ -5,19 +5,25 @@ pub use input::*;
 
 use std::io::{self, Read, Write};
 
-use ansi_term::Color;
 use ansi_term::Color::Fixed;
+use ansi_term::{Color, Style};
 
 use crate::squeezer::{SqueezeAction, Squeezer};
 
 const BUFFER_SIZE: usize = 256;
 
-const COLOR_NULL: Color = Fixed(242); // grey
-const COLOR_OFFSET: Color = Fixed(242); // grey
-const COLOR_ASCII_PRINTABLE: Color = Color::Cyan;
-const COLOR_ASCII_WHITESPACE: Color = Color::Green;
-const COLOR_ASCII_OTHER: Color = Color::Purple;
-const COLOR_NONASCII: Color = Color::Yellow;
+lazy_static::lazy_static! {
```

Comment:
There is a more modern alternative to `lazy_static` called [`once_cell`](https://docs.rs/once_cell/latest/once_cell/). Maybe we could use it here? (See also: https://docs.rs/once_cell/latest/once_cell/#faq)

Reply:
Done in 61aea39165a7a798f2ab6650f66cfdbf2887c514